### ISSUE TITLE
Architecture diagrams in the docs

### ISSUE THREAD

**Issue by flaki** [suborbital/docs#20]
We have a couple architecture diagrams in the Suborbital Miro workspace which would make sense to be exposed in the new docs. We should talk about what format would make sense (initially we can start with simple static SVG-s with perhaps some accompanying explanatory text, but later on more interactive/animated visualizations could be useful too)

**Comment by LauraLangdon** [comment]
I this issue also encompasses the remaining part of Issue #2.

**Comment by flaki** [comment]
Correct, I wanted to use our internal diagrams to make some illustrations to spice up the docs but haven't yet been able to get around to this.
